libfuse/folderlist.go

```diff
@@ -31,11 +31,24 @@ type FolderList struct {
 	recentlyRemoved   map[libkbfs.CanonicalTlfName]bool
 }
 
+var _ fs.NodeAccesser = (*FolderList)(nil)
+
+// Access implements fs.NodeAccesser interface for *FolderList.
+func (*FolderList) Access(ctx context.Context, r *fuse.AccessRequest) error {
+	if int(r.Uid) != os.Getuid() {
+		// short path: not accessible by anybody other than the logged in user.
+		// This is in case we enable AllowOther in the future.
+		return fuse.EPERM
+	}
+
+	return nil
```

Comment:
You should return an error for anything asking about write permissions in the folderlist.